assets/components/src/category-autocomplete/index.js

```diff
@@ -107,21 +121,24 @@ class CategoryAutocomplete extends Component {
 			label,
 			value,
 		} = this.props;
-		const { suggestions, allCategories } = this.state;
+		const { allCategories } = this.state;
 		const classes = classnames( 'newspack-category-autocomplete', className );
 		return (
 			<div className={ classes }>
 				<FormTokenField
 					onInputChange={ input => this.debouncedUpdateSuggestions( input ) }
 					value={ value.reduce( ( acc, item ) => {
-						const category =
+						const categoryOrItem =
```

Comment:
Non-blocking suggestion: normalize the `categoryOrItem` object in a separate function that ensures the return of `id` and `value` for better readability and maintainability.

Reply:
Good idea, but I'm not _entirely_ sold since I don't see how it could be reused. I'll leave it as it is for now.